packages/foreman/foreman/foreman.spec

```diff
@@ -672,6 +672,7 @@ Meta Package to install dynflow sidekiq executor support
 %systemd_post %{dynflow_sidekiq_service_name}.service
 
 %preun dynflow-sidekiq
+%systemd_preun dynflow.service
```

Comment:
It used to be part of the main foreman package, not `dynflow-sidekiq`. Upgrading users won't have that installed yet. IIRC it was also `dynflowd` and not `dynflow`.